### PR TITLE
Create README.md

### LINKED ISSUE #15: Creating a README
As a software developer working on an open source project, the audience need to know what a list of things about the project.

**Acceptance Criteria**
Defining the following:
- What this project does?
- Why this project is useful?
- How to started?
- Where to get more help, if needed?

### PR DESCRIPTION
**Pull request by rishab-goswami**
resolves #15 
- [x] The pull request is complete according to the following criteria:
  - [ ] Acceptance criteria have been met
  - [ ] The documentation is kept up-to-date
  - [ ] Comprehensive tests (if applicable) have been generated and all pass.
  - [ ] The pull request describes the changes that have been made, and enough information is present in the description for any developer to understand what has changed
  - [ ] Commits have been squashed (or will be on merge).
  - [ ] The branch name is descriptive and follows the pull request title format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [ ] The pull request title is of the following format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [ ] The description uses [github syntax](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) to link to the issue. E,g Resolves se701g2/Doto#{Number}
  - [ ] At least two reviewers assigned. One of which must be the assigner of the issue.
  - [ ] If there are merge conflicts, run git rebase as opposed to git merge with master.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+5) Test changes
+6) Create a pull request
+7) The pull request must be reviewed and approved by at least 2 other members
+8) Once the reviewers approve the pull request, they can merge your contributions into the main repository. DO NOT MERGE YOUR OWN PULL REQUESTS
+9) For more details about the exactly git practices and conventions check out our GIT wiki
+
+## Meta
+Se701 Group2 - Se701group2@gmail.com
+Distributed under the MIT licence. Check the Wiki for more details
+https://github.com/se701g2
```
Comment (lakshyaoberoi) [inline README.md]: Make the URL point to the Wiki page

File: README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+## Development Setup
+It is recommended that you use VsCode when contributing to this project. Please install the Eslint and Prettier extensions so that the code style matches what has already been done before.
+
+## Running the code
+We are using Npm for this particular project (https://www.npmjs.com/get-npm).
+```bash
+Npm install
+```
+```bash
+Npm start
```
Comment (MarioSinovcic) [inline README.md]: it's "Npm" it's "npm" right?

also I would recommend that you write "cd doto-frontend" or "cd doto-backend" or else people will end up creating packages in the root directory.

also "npm run start" not "npm start" right?

File: README.md (hunk 1 above)
Comment (daltonlim) [reply README.md]: ^ Probably add a reference to the wiki to get more information on the practices we have.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+## Running the code
+We are using Npm for this particular project (https://www.npmjs.com/get-npm).
+```bash
+Npm install
+```
+```bash
+Npm start 
+```
+
+Please check the front end readme for more information on running the front end of the code.
```
Comment (daltonlim) [inline README.md]: Add link to readme if possible

File: README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## Usage Examples
+(will do once more of the website is done)
+
+## Contributing to this Project
```
Comment (daltonlim) [inline README.md]: Link to wiki. No need to repeat practices

File: README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+## Why is this project Useful?
+This project is useful as it has the basic functionalities of any normal calendar app, but also has the added functionality of smart scheduling. This is particularly useful for people who like to plan their day but find it difficult to plan it well, as the app finds the most suitable time for whatever task the user wants to accomplish. It also encourages people who don’t usually like to plan their day to start planning their day as all they need to do is input the task and the app will choose the perfect time, no effort required. 
+
+
+## Development Setup
+It is recommended that you use VsCode when contributing to this project. Please install the Eslint and Prettier extensions so that the code style matches what has already been done before.
```
Comment (daltonlim) [inline README.md]: Maybe highlight the prerequisites

File: README.md (hunk 2 above)
Comment (rishab-goswami) [reply README.md]: I checked the front end readme and it says "npm start". Not sure if there is a difference.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+## Usage Examples
+(will do once more of the website is done)
+
+## Contributing to this Project
+Please refer to our [wiki] (https://github.com/se701g2/Doto/wiki) for an outline of our coding conventions and git workflow
+
+## Meta
+Se701 Group2 - Se701group2@gmail.com
+Distributed under the MIT licence. Check the [wiki] (https://github.com/se701g2/Doto/wiki/licence) for more details
```
Comment (daltonlim) [inline README.md]: Spelling

File: README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+Please refer to the [wiki] (https://github.com/se701g2/wiki) for our coding practices
```
Comment (daltonlim) [inline README.md]: Could link directly to https://github.com/se701g2/Doto/wiki/Git-Practices-and-Conventions. Also is this a repeat of above?

File: README.md [hunk 8]
```diff
@@ -7,37 +7,29 @@ This project is useful as it has the basic functionalities of any normal calenda
...
-Npm start 
-```
+We are using npm for this particular project (https://www.npmjs.com/get-npm).
+### `cd doto-frontend/backend`
+(change to either frontend or backend depending on where you wish to contribute)
 
-Please check the front end readme for more information on running the front end of the code.
+### `npm install`
+
+### `npm start`
```
Comment (daltonlim) [inline README.md]: Is there a reason for making the code headers.

Comment (rishab-goswami) [reply README.md]: Um, It looks nice I guess. If you really don't want it then I can take it out

File: README.md (hunk 7 above)
Comment (rishab-goswami) [reply README.md]: I just took that out

File: README.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+## Usage Examples
+(will do once more of the website is done)
+
+## Contributing to this Project
+Please refer to our [wiki] (https://github.com/se701g2/Doto/wiki) for an outline of our coding conventions and git workflow
+
+## Meta
+Se701 Group2 - Se701group2@gmail.com
+Distributed under the MIT license. Check the [wiki] (https://github.com/se701g2/Doto/wiki/licence) for more details
```
Comment (daltonlim) [inline README.md]: Hi, the url is broken but overall great work 💯

**Pull request merged**